_about_livecoms/03_earlycareer.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: single
+sidebar:
+  nav: earlycareer.md
+title: For Early Career Scientists 
```

Comment:
I wonder whether students will realize this is for them.  What about "For sudents and early-career scientists"